CMakeLists.txt

```diff
@@ -306,6 +308,24 @@ if(BUILD_TESTING)
   find_package(Threads REQUIRED)
   target_link_libraries(ninja_test PRIVATE libninja libninja-re2c GTest::gtest Threads::Threads)
 
+  # Use RadpidJSON on Windows to support MSVC's /sourceDependencies format.
+  if(WIN32)
+    find_package(RapidJSON)
+    if(NOT RapidJSON_FOUND)
+      include(FetchContent)
+      FetchContent_Declare(
+        rapidjson
+        URL https://github.com/Tencent/rapidjson/archive/f56928de85d56add3ca6ae7cf7f119a42ee1585b.zip
```

Comment:
Any reason not to use an official release archive, e.g. https://github.com/Tencent/rapidjson/archive/refs/tags/v1.1.0.zip instead?

Reply:
The latest release tag is from 2016.   This is a newer version.

Unfortunately the latest `master` version doesn't seem compatible with `FetchContent`, so this version is a compromise for now.

Reply:
After vendoring RapidJSON headers in https://github.com/ninja-build/ninja/pull/2600#discussion_r2110027264, this is moot.